pipelines/tekton/aiedge-e2e/examples/tensorflow-housing.trigger-template.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: triggers.tekton.dev/v1beta1
+kind: TriggerTemplate
+metadata:
+  name: tensorflow-housing
+spec:
+  params:
+  - name: model-name
+    default: tensorflow-housing
+  - name: model-version
+    default: "1"
+  - name: s3-bucket-name
+    default: rhoai-edge-models
+  - name: git-containerfile-repo
+    default: https://github.com/opendatahub-io/ai-edge.git
+  - name: git-containerfile-revision
+    default: "main"
+  - name: containerfileRelativePath
+    default: pipelines/containerfiles/Containerfile.openvino.mlserver.mlflow
+  - name: fetch-model
+    default: "git"
+  - name: git-model-repo
+    default: https://github.com/opendatahub-io/ai-edge.git
+  - name: modelRelativePath
+    default: pipelines/models
+  - name: model-dir
+    default: "tf2model"
+  - name: git-model-revision
+    default: "main"
+  - name: test-endpoint
+    default: "v1/models/tensorflow-housing/versions/1:predict"
+  - name: candidate-image-tag-reference
+    default: image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(params.model-name):$(params.model-version)-candidate
+  - name: target-image-tag-references
+    default: quay.io/grdryn/$(params.model-name):$(params.model-version)-$(context.pipelineRun.uid)
+    #- quay.io/grdryn/$(params.model-name):$(params.model-version)
+    #- quay.io/grdryn/$(params.model-name):latest
```

Comment:
TriggerTemplate params seemingly need to have string defaults, See L77 where I've converted that string back to a list for the PipelineRun parameter.